src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -130,5 +130,27 @@ class Bar : I2
             await VerifyItemExistsAsync(markup, "Goo2()");
             await VerifyItemExistsAsync(markup, "Prop");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ExplicitInterfaceMemberCompletionIgnoresPropertyGetterOrSetterMethod()
+        {
+            var markup = @"
+interface i1
+{
+    int Prop { get; set;}
+
+    int TestMethod(); 
+}
+
+public class MyClass : i1
+{
+    int i1.$$
+}
+";
+
+            var completionItems = await GetCompletionItemsAsync(markup, SourceCodeKind.Regular);
+            Assert.DoesNotContain(completionItems, ci => ci.DisplayText == "Prop.get" || ci.DisplayText == "Prop.set");
```

Comment:
tests for events with accessors would also be good.

Reply:
There should be a method for this, something similar to `VerifyItemIsAbsentAsync`. Does it not exist? If so, can you please use it here?

Reply:
First, good catch @CyrusNajmabadi because events had the same issue. @Neme12 - I switched to using VerifyItemIsAbsentAsync as suggested. I removed this unit test because it seemed redundant since there was a very similar unit test already. I left a comment addressing the test I extended.